x-pack/plugins/security_solution/public/resolver/store/middleware/related_events_fetcher.ts

```diff
@@ -65,47 +73,43 @@ export function RelatedEventsFetcher(
           });
         }
       } catch (error) {
-        api.dispatch({
-          type: 'serverFailedToReturnNodeEventsInCategory',
-          payload: {
-            nodeID,
-            eventCategory,
-            cursor,
-          },
-        });
+        api.dispatch(
+          serverFailedToReturnNodeEventsInCategory({ id, nodeID, eventCategory, cursor })
+        );
       }
 
       if (result) {
-        api.dispatch({
-          type: 'serverReturnedNodeEventsInCategory',
-          payload: {
+        api.dispatch(
+          serverReturnedNodeEventsInCategory({
+            id,
             events: result.events,
             eventCategory,
             cursor: result.nextEvent,
             nodeID,
-          },
-        });
+          })
+        );
       }
     }
 
     // If the panel view params have changed and the current panel view is either `nodeEventsInCategory` or `eventDetail`, then fetch the related events for that nodeID.
+    // console.debug(oldParams, newParams);
     if (!isEqual(newParams, oldParams)) {
       if (newParams.panelView === 'nodeEventsInCategory') {
         const nodeID = newParams.panelParameters.nodeID;
-        api.dispatch({
-          type: 'appRequestedNodeEventsInCategory',
-          payload: {
-            parameters: newParams,
-          },
-        });
+        // api.dispatch(
```

Comment:
@kqualters-elastic there was no reducer for `appRequestedNodeEventsInCategory`, and I'm not sure passing parameters do anything as we grab params from url now?

Should we pass some `pendingRequest` as safeguard?